src/Features/VisualBasic/Portable/AddImport/VisualBasicAddImportCodeFixProvider.vb

```diff
@@ -99,11 +85,33 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.AddImport
         ''' </summary>
         Friend Const BC30182 = "BC30182"
 
-        Public Overrides ReadOnly Property FixableDiagnosticIds As ImmutableArray(Of String)
+        Public ReadOnly Property FixableDiagnosticIds As ImmutableArray(Of String)
             Get
                 Return ImmutableArray.Create(BC30002, BC30451, BC30456, BC32042, BC36593, BC32045, BC30389, BC31504, BC32016, BC36610,
                                              BC36719, BC30512, BC30390, BC42309, BC30182, IDEDiagnosticIds.UnboundIdentifierId)
             End Get
         End Property
+    End Module
+
+    <ExportCodeFixProvider(LanguageNames.VisualBasic, Name:=PredefinedCodeFixProviderNames.AddImport), [Shared]>
+    Friend Class VisualBasicAddImportCodeFixProvider
+        Inherits AbstractAddImportCodeFixProvider
+
+        Public Sub New()
+        End Sub
```

Comment:
can be removed

Reply:
Actually the empty constructor is the normal way this provider is used and is needed.

Reply:
oh, i see.  there's another constructor as well.  👍